torch/_dynamo/convert_frame.py

```diff
@@ -477,6 +477,10 @@ def log_bytecode(prefix, name, filename, line_no, code):
         )
 
         assert output is not None
+
+        if output.graph.is_empty_graph():
```

Comment:
Please add a comment why this is required (and specifically why it was causing issues for export).
It could be something like this

Skipping Dynamo on a frame without any extracted graph. This does not affect eager functionality. But this is necessary for export for cases where Dynamo-reconstructed bytecode can create new function frames, confusing export in thinking that there are extra graphs now.